src/components/landing/LoginCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+
+function LoginCard() {
+  async function login(event) {
+    event.preventDefault();
+
+    // TODO API
+  }
+
+  return (
+    <LandingCard title="Welcome Back">
+      <form onSubmit={login}>
+        <TextInput placeHolder="username" key="Username" />
+        <TextInput placeHolder="password" key="Password" type="password" />
```

Comment:
Could you capitalize `username` and `password` for the placeholder? That's just to match the capitalization in the other PRs.